applications/matter_bridge/Kconfig.sysbuild

```
@@ -33,7 +38,7 @@ config PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY
 if SOC_SERIES_NRF53X
 
 config MCUBOOT_UPDATEABLE_IMAGES
-	default 3 if BOARD_NRF5340DK_NRF5340_CPUAPP #### Support for WiFi patch on external flash memory.
+	#default 3 if BOARD_NRF5340DK_NRF5340_CPUAPP #### Support for WiFi patch on external flash memory.
```

We should not remove it :) I think we should do something like 
```suggestion
	default 3 if (BOARD_NRF5340DK_NRF5340_CPUAPP && WIFI_NRF700X) #### Support for WiFi patch on external flash memory.
```